src/controllers/movieController.js

```diff
@@ -0,0 +1,51 @@
+const { Prisma } = require("@prisma/client");
```

Comment:
naming this and the screen controller files differently to the rest is a little jarring, you could maintain a consistent naming convention across the app so it is easier to follow